tailwind.config.js

```diff
@@ -17,6 +17,9 @@ export default {
           '2xl': '183px',
         },
       },
+      fontSize: {
+        '2rem': '2rem',
```

Comment:
Why?

Reply:
I checked the following PRs, are you going to use it as text-2rem?

Reply:
Yes, TW doesn't have 2rem. And I have many elements with 2rem and it's really look better with 2rem (not 1.1875 ot 2.25rem). 
Or do u think that name 2rem is not a good idea?
![image](https://github.com/voyageglobal/voyage-frontend/assets/170460349/d7b8ac7c-e6f5-434d-be8c-41efa6151557)

Reply:
Let's leave it as it is